weave/panels_py/panel_openai_monitor.py

```diff
@@ -110,7 +110,20 @@
 
 
 def cost(row: dispatch.RuntimeOutputNode) -> dispatch.RuntimeOutputNode:
-    return row["summary.total_tokens"] * 0.0015e-3
+    return weave.ops.cond(
```

Comment:
what happens of the model is neither?

Reply:
Then you would get None. In the final version of the cost function, I use a final "True" case to return 0 for now.